packages/indy-sdk-to-askar-migration/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@aries-framework/indy-sdk-to-askar-migration",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.3.3",
+  "files": [
+    "build"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/indy-sdk-to-askar-migration",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/indy-sdk-to-askar-migration"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aries-framework/anoncreds": "0.4.0-alpha.71",
+    "@aries-framework/askar": "0.4.0-alpha.71",
+    "@aries-framework/core": "0.4.0-alpha.71",
+    "@aries-framework/indy-sdk": "0.4.0-alpha.71",
+    "@aries-framework/node": "0.4.0-alpha.71",
+    "@hyperledger/aries-askar-shared": "file:/Users/beri/Developer/work/hyperledger/aries-askar/wrappers/javascript/aries-askar-shared"
+  },
+  "devDependencies": {
+    "@hyperledger/aries-askar-nodejs": "file:/Users/beri/Developer/work/hyperledger/aries-askar/wrappers/javascript/aries-askar-nodejs",
+    "indy-sdk": "1.16.0-dev-1655",
```

Comment:
It's needed to fix the version to 1655 or can be aligned with other modules that have ^1655?

Reply:
Yeah I am fine with that. Personally, I just always lock them to a specific version as I have encountered my times than I would like to admit that a breaking change is pushed on a patch version. So I just did it out of habit here :).